lib/tasks/publishing_api.rake

```diff
@@ -0,0 +1,11 @@
+namespace :publishing_api do
+  task :republish_content => [:environment] do
+    puts "Scheduling republishing of #{Edition.published.count} editions"
+
+    Edition.published.each do |edition|
```

Comment:
Mongo :disappointed: 

Mongoid 3.X has pluck, but we're on 2.5. https://github.com/mongodb/mongoid/blob/master/CHANGELOG.md#310